tests/test_blockscout.py

```diff
@@ -123,3 +123,11 @@ def test_get_contract_type_with_rate_limiting(mock_backend, explorer):
 def test_too_many_requests_error(no_api_key, response):
     actual = str(BlockscoutTooManyRequestsError(response, "ethereum"))
     assert "BLOCKSCOUT_API_KEY" in actual
+
+
+@pytest.mark.parametrize("ecosystem", ["base", "ethereum", "gnosis", "optimism", "polygon"])
```

Comment:
for a more dynamic test against future additions, we could import and use keys from https://github.com/ApeWorX/ape-blockscout/blob/main/ape_blockscout/utils.py#L11